src/cargo/ops/mod.rs

```diff
@@ -28,7 +30,7 @@ pub use self::resolve::{
 pub use self::vendor::{vendor, VendorOptions};
 
 mod cargo_clean;
-mod cargo_compile;
+pub mod cargo_compile;
```

Comment:
This shouldn't be `pub`.  `unit_dependencies` should import with `use crate::ops::resolve_all_features;`.

Reply:
Handled.